trl/trainer/kto_trainer.py

```diff
@@ -313,9 +313,16 @@ def make_inputs_require_grad(module, input, output):
         self.undesirable_weight = args.undesirable_weight
 
         # get KL datasets
-        train_KL_dataset = train_dataset.map(self.get_KL_dataset, batched=True, batch_size=1000)
+        total_batch_size = torch.cuda.device_count() * args.per_device_train_batch_size
```

Comment:
Would this be a more precise definition of total batch size?
```suggestion
        total_batch_size = torch.cuda.device_count() * args.per_device_train_batch_size * args.gradient_accumulation_steps
```